### PR TITLE
update email_service to work with data containing website

### LINKED ISSUE #98: update email_service to work with data containing website
Update the `email_service.py` to use `website_name` in the emails!

### PR DESCRIPTION
**Pull request by BerkayDur**
Updates the email pipeline to take into consideration the website name.

Closes #98

### CODE REVIEW

File: ETL/Email/email_service.py [hunk 1]
```diff
@@ -138,7 +136,7 @@ def format_email_from_data_frame(
         if not email_type:
             email_type = 'sale'
 
-    message = f"({website_name}) <a href='{row_data['url']}'>{row_data['product_name']}</a> "
+    message = f"({row_data['website_name']}) <a href='{row_data['url']}'>{row_data['product_name']}</a> "
```
Comment (A-Faris) [inline ETL/Email/email_service.py]:
```suggestion
    message = f"({row_data['website_name']}) \
    <a href='{row_data['url']}'>{row_data['product_name']}</a> "
```

File: ETL/Email/email_service.py [hunk 2]
```diff
@@ -315,4 +313,4 @@ def send_emails(
     load_dotenv('.env')
     connection_obj = get_connection(CONFIG)
     client = get_ses_client(CONFIG)
-    send_emails(connection_obj, client, [[{}, {}]], PRODUCT_READING_KEYS)
+    send_emails(connection_obj, client, [{}], PRODUCT_READING_KEYS)
```
Comment (A-Faris) [inline ETL/Email/email_service.py]: Add empty line

File: ETL/Email/email_service.py
Comment (A-Faris) [inline ETL/Email/email_service.py:1]: Just basic pylint issues to fix in this file

**Review by A-Faris — DISMISSED** [review]
Just pylinting is needed to make it perfect

File: ETL/Email/email_service.py (hunk 1 above)
Comment (BerkayDur) [reply ETL/Email/email_service.py]: Good spot!

**Review by A-Faris — DISMISSED** [review]
Good changes

**Review by A-Faris — DISMISSED** [review]
Good work

**Review by A-Faris — DISMISSED** [review]
Good work again

**Review by NabihaMoh — APPROVED** [review]
Looks good.

**Pull request merged**